.github/workflows/test-all-packages.yml

```diff
@@ -4,9 +4,10 @@ name: Test all Packages
 # branches)
 
 on:
- push:
-   branches: [ $default-branch ]
- pull_request:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    types: [opened, closed, reopened, synchronize]
```

Comment:
what type of PR does this omit?

Reply:
Actually, adds `closed` (which is not triggered by default) to the default list.  I'll add a comment referring to Github documentation.